.github/workflows/ios-e2e-test.yml

```diff
@@ -42,3 +42,8 @@ jobs:
       - name: Test app
         working-directory: ${{ env.WORKING_DIRECTORY }}
         run: yarn test-e2e-ios
+      - uses: actions/upload-artifact@v2
+        if: ${{ failure() }}
+        with:
+          name: android-fail-screen-shoots
```

Comment:
Why is there `android` if the PR mentions `iOS`?

Reply:
A copy paste typo 🤦